iquip/apps/builder.py

```diff
@@ -365,21 +355,112 @@ def get_state(self, argInfo: Dict[str, Any]) -> Dict[str, Any]:
                     state[ty] = default
         return state
 
-    def get_procdesc(self, argInfo: Dict[str, Any]) -> Dict[str, Any]:
-        """Gets a procdesc dictionary that describes common parameters of the scannable object.
+    def get_procdesc(self) -> Dict[str, Any]:
+        """Gets a procdesc dictionary that describes common parameters of the scannable object."""
+        procdesc = {
+            "unit": self.argInfo["unit"],
+            "scale": self.argInfo["scale"],
+            "global_step": self.argInfo["global_step"],
+            "global_min": self.argInfo["global_min"],
+            "global_max": self.argInfo["global_max"],
+            "ndecimals": self.argInfo["ndecimals"]
+        }
+        return procdesc
+
+    def value(self) -> Dict[str, Any]:
+        """Overridden.
+        
+        Returns the dictionary of the selected scannable arguments.
+        """
+        selected = self.state["selected"]
+        return self.state[selected]
+
+
+class _RangeScan(QWidget):
+    """A widget for range scan in _ScanEntry.
+
+    Attributes:
+        scale: The scale factor that is multiplied to the number value.
+        state: Each key and its value as follows.
```

Comment:
As I understand, we don't have to make `state` an attribute anymore, because we won't track it.
What do you think of it?

Reply:
correct.